connector/src/main/java/org/jboss/as/connector/subsystems/datasources/Constants.java

```diff
@@ -457,7 +457,7 @@ public void validateResolvedParameter(String parameterName, ModelNode value) thr
             .setXmlName(Security.Tag.AUTHENTICATION_CONTEXT.getLocalName())
             .setAllowExpression(true)
             .setRequires(ELYTRON_ENABLED_NAME)
-            .addAlternatives(SECURITY_DOMAIN_NAME, USERNAME_NAME)
+            .addAlternatives(SECURITY_DOMAIN_NAME)
```

Comment:
nm, I see that is alternatives set on username attribute

.addAlternatives(SECURITY_DOMAIN_NAME, AUTHENTICATION_CONTEXT_NAME)